dev/README.md

```diff
@@ -22,38 +22,18 @@
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 **Table of contents**
 
-- [Airflow Jira utility](#airflow-jira-utility)
-- [Airflow Pull Request Tool](#airflow-pull-request-tool)
-- [Airflow release signing tool](#airflow-release-signing-tool)
+  - [Airflow Pull Request Tool](#airflow-pull-request-tool)
```

Comment:
Can you keep a heading hierarchy?
https://developers.google.com/style/headings

Reply:
Ah indeed we miss the first heading

Reply:
The develpment tools one was above the TOC that 's why it was missing from the automatically generated TOC

Reply:
Should be much nicer now :)